documentation/src/main/resources/templates/downloadlocations.ftl

```diff
@@ -4,8 +4,10 @@ The following are download locations for the current version of [detect_product_
 
 * [detect_product_short] Bash script: [Bash script](${script_repo_url_bash})
 * The [detect_product_short] PowerShell script: [PowerShell script](${script_repo_url_powershell})
-* The [detect_product_short] binary repository (.jar and air gap zip files): [Binary files](${binary_repo_url_project})
-* The [detect_product_short] repository for the Jenkins plugin: [Release Artifacts Repository](${binary_repo_jenkins_url_project})
+* The [detect_product_short] binary repository (.jar and air gap zip files) for 10.0.0 and later: [Binary files](${binary_repo_url_project})
+* The [detect_product_short] binary repository (.jar and air gap zip files) for 9.10.1 and earlier: [Binary files](${binary_repo_url_project_old})
+* The download location for the [detect_product_long] Jenkins plugin: [Jenkins Plugins](https://plugins.jenkins.io/blackduck-detect/)
```

Comment:
@cpottsbd Jenkins plugin is not downloaded like binaries. Change this "The download location for the Black Duck® Detect Jenkins plugin:" to "Black Duck® Detect Jenkins plugin is available here: [Jenkins plugin]" (no need for plural here I think for Plugins!)

Reply:
True enough.